src/service/utils/layers.js

```diff
@@ -92,14 +92,17 @@ export const handleSelectQuality = (media) => {
   if (!selectedData.encodingId && media.spatialLayerId !== null) {
     selectedData.spatialLayerId = parseInt(media.spatialLayerId)
   }
-  const data =
-    selectedData.encodingId ||
-    selectedData.encodingId === 0 ||
-    selectedData.spatialLayerId ||
-    selectedData.spatialLayerId === 0
-      ? selectedData
-      : {}
-  state.ViewConnection.millicastView.select(data)
+  const source = state.Sources.selectedVideoSource
```

Comment:
@juanpiquerez can you add a comment as to why we've changed from `select` to `project` so people don't revert in the future?

Reply:
Yes, sure, thanks!